template/cypress/cypress-e2e.json

```diff
@@ -0,0 +1,10 @@
+{
+    "projectId": "rqdhjf2",
```

Comment:
This id is related to a project in cypress dashboard. It should be removed from template